packages/@orchidui-vue/src/Elements/AdditionalContent/OcAdditionalContent.vue

```diff
@@ -53,16 +53,16 @@ const copyLink = async () => {
       :title="additionalTitle"
       class="border rounded md:py-4 py-3 md:px-5 px-4 border-oc-gray-200 bg-oc-gray-50 flex flex-col gap-y-2"
     >
-      <template #title="{ title }">
+      <template #title>
```

Comment:
If the slot isn't receiving the `title` prop then it would be good to remove passing it through in `OcTitle` to avoid confusion in future.

Reply:
`title` prop is required in `ocTitle` component. So I kept it, even tho ocAddittionalContent component is overriding the slot.

Reply:
Not the prop on `OcTitle` itself. I meant the prop being passed into the slot:
`<slot name="title" :title="title">`

Reply:
ocTitle component has been updated and doesn't support scoped slots, so I removed the slot props.

Reply:
Sorry my bad. Was looking at an outdated version of OcTitle